bundles/org.openhab.core.automation.rest/src/main/java/org/openhab/core/automation/rest/internal/RuleResource.java

```diff
@@ -121,6 +126,15 @@ public RuleResource( //
         this.ruleRegistry = ruleRegistry;
     }
 
+    @Reference(cardinality = ReferenceCardinality.OPTIONAL, policy = ReferencePolicy.DYNAMIC)
```

Comment:
Is this really an optional reference? Isn't it mandatory and we can pass it via constructor injection?

Reply:
Agree, can be mandatory.